### PR TITLE
Add basic UI

### PR DESCRIPTION
**Pull request by BenMukebo**
## In this Milestone, I create a form of UI for the school library app. This way it can be invoked as an executable and not something you use in IRB exclusively.

- Create a main.rb file that will serve as console app entry-point. It should have methods that do the following:
   - List all books.
   - List all people.
   - Create a person (teacher or student, not a plain Person).
   - Create a book.
   - Create a rental.
   - List all rentals for a given person id.
- In the main.rb define the entry point, this will be a method called main that is invoked at the end of your file.
-  This method should do the following:
   - Present the user with a list of options to perform.
   - Lets users choose an option.
   - If needed, ask for parameters for the option.
   - Have a way to quit the app.

### CODE REVIEW

File: main.rb [hunk 1]
```diff
@@ -0,0 +1,196 @@
+#!/usr/bin/env ruby
+require 'pry'
```
Comment (dasileker) [inline main.rb]: - [ ] Please remove this it gives an error while executing.

File: student.rb [hunk 2]
```diff
@@ -1,17 +1,33 @@
+# require_relative('./person')
+
+# class Student < Person
+#   # attr_reader :classroom
+
+#   def initialize( age, name: 'Unknown', parent_permission: true, classroom: 'Unknown')
+#     super(name, age, parent_permission)
+#     @classroom = classroom
```
Comment (dasileker) [inline student.rb]: - [ ] Please remove all commented codes.

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @BenMukebo,** 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: student.rb (hunk 2 above)
Comment (BenMukebo) [reply student.rb]: Sure, thanks Dasileker

File: main.rb (hunk 1 above)
Comment (BenMukebo) [reply main.rb]: Thanks

**Review by bimbolabuari — APPROVED** [review]
# Project Approved 🚀

Hi @BenMukebo 🥇,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**